transfer_learning.ipynb

```diff
@@ -317,13 +294,14 @@
     "                    if phase == 'train':\n",
     "                        loss.backward()\n",
     "                        optimizer.step()\n",
+    "                        scheduler.step()\n",
```

Comment:
There was a change in PyTorch 1.1 that now makes the scheduler go after the first `optimizer.step()`, see https://github.com/pytorch/pytorch/pull/7889

Reply:
annnd, this introduced a bug :-)

It should happen once per epoch, and not once every iteration. Sorry about the bug!